assets/css/frontend.scss

```diff
@@ -455,6 +455,10 @@ a.sensei-certificate-link {
 
 	.sensei-quiz-action {
 		margin: 0;
+		padding: 0;
+		button.button {
+			padding: 8px 11px;
```

Comment:
I think this will also change the button padding outside of the course theme. Is this intentional?

Reply:
Nice catch! I put it away into course-theme related CSS. 9d5360a3562dacdb5f96b5e81d007b032bd889ce